### DISCUSSION TITLE
SW360 project and EclipseSW360 project.

### DISCUSSION THREAD

**Discussion by KoukiHama** [eclipse-sw360/sw360#1974] [General]
There are two SW360-related projects on GitHub. This causes some confusion.

There is no need to combine them into one, but it may be necessary to add a clear explanation.

https://github.com/eclipse-sw360/

https://github.com/sw360/

**Comment by arunazhakesan** [comment]
I guess it is 2 different GitHub orgs rather than 2 projects. It would be useful to add some information to clarify this.
